README.md

```diff
@@ -261,24 +269,25 @@ interface XRWebGLGraphicsBinding {
   double getNativeProjectionScaleFactor();
 
   Promise<XRProjectionLayer> requestProjectionLayer(XRWebGLLayerInit init); // Note different dictionary
-  Promise<XRQuadLayer> requestQuadLayer(XRWebGLLayerSourceInit init);
-  Promise<XRCylinderLayer> requestCylinderLayer(XRWebGLLayerSourceInit init);
-  Promise<XREquirectLayer> requestEquirectLayer(XRWebGLLayerSourceInit init);
-
-  WebGLFramebuffer? getCurrentFramebuffer(XRLayer layer);
+  Promise<XRQuadLayer> requestQuadLayer(XRLayerInit init);
+  Promise<XRCylinderLayer> requestCylinderLayer(XRLayerInit init);
+  Promise<XREquirectLayer> requestEquirectLayer(XRLayerInit init);
+  
+  XRSubImage? getViewSubImage(XRLayer layer); // for mono layers
+  XRSubImage? getViewSubImage(XRLayer layer, XRView view); // for stereo layers
```

Comment:
Oh! I missed earlier that you moved these methods off of the layer types and onto here. If that's the case, we can have them return the appropriate `XRSubImage` type directly, which removes some of the opaqueness. 👍